test/lazy/test_kronecker_product_added_diag_lazy_tensor.py

```diff
@@ -43,6 +41,13 @@ def evaluate_lazy_tensor(self, lazy_tensor):
         diag = lazy_tensor._diag_tensor._diag
         return tensor + diag.diag()
 
+class TestKroneckerProductAddedConstDiagLazyTensor(TestKroneckerProductAddedDiagLazyTensor):
+    should_call_lanczos = False
+    should_call_cg = False
+
+    def create_lazy_tensor(self):
+        return super().create_lazy_tensor(constant_diag=True)
+
     def test_if_cholesky_used(self):
```

Comment:
should these not be defined in `TestKroneckerProductAddedDiagLazyTensor` ?

Reply:
yes.